addons/binding/org.openhab.binding.sonos/OSGI-INF/.gitignore

```diff
@@ -1 +0,0 @@
-/*.xml
```

Comment:
If I understood it correctly from the webui repo the OSGI-INF is/was still generated so the .gitignore was still needed for Eclipse, is that also the case here? Or does that not apply any more?

Reply:
I figured it out later that was due to the fact the Mavenized projects had to be deleted and reimported into Eclipse. After that Eclipse doesn't generate these files anymore. :smile:

Reply:
You figured out that I missed to do that step once.
I need to import the project and add the generated Eclipse IDE files again.
Will add another commit later.